dtests/src/test/scala/io/snappydata/hydra/ct/CTTestUtil.scala

```diff
@@ -214,7 +214,8 @@ object CTTestUtil {
   Load data to already created tables by removing duplicate records.
  */
   def loadTablesByRemovingDuplicateRecords(snc: SnappyContext): Unit = {
-    CTQueries.orders_details_df(snc).dropDuplicates("SINGLE_ORDER_DID").write.insertInto("orders_details")
+    CTQueries.orders_details_df(snc).dropDuplicates("SINGLE_ORDER_DID")
```

Comment:
Please remove formatting changes in this method. I have modified the code for fixing a test issue with concurrent putInto.